src/applications/disability-benefits/view-payments/components/view-payments-lists/payments/Payments.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import Pagination from '@department-of-veterans-affairs/formation-react/Pagination';
+import ResponsiveTable from '../../responsive-table/ResponsiveTable';
+import { clientServerErrorContent } from '../helpers';
+import { chunk } from 'lodash';
+
+const alertClasses =
+  'vads-u-padding-y--2p5 vads-u-padding-right--4 vads-u-padding-left--2';
+
+class PaymentsReceived extends Component {
+  state = {
+    page: 1,
+    maxRows: 5,
+    numberOfPages: null,
+    currentlyShowingData: [],
+    paginatedData: null,
+    fromNumber: null,
+    toNumber: null,
```

Comment:
Is this always going to be stored locally in this component, or is there intent to move some of this data to redux? In dealing with a larger number of pages, it might be worthwhile to store the page they are on in redux, so on browser refresh they can keep their page state instead of being sent back to the beginning.

Reply:
I intend to refactor this into Redux eventually, just not in this ticket.